ionic_user_interface/src/views/Home.vue

```diff
@@ -2,9 +2,12 @@
   <ion-page>
     <ion-content :fullscreen="true">
       <div id="container">
-        <strong>Route Maker</strong>
-        <p>Quickly make custom climbing routes</p>
-        <br />
+        <div v-if="!photoData">
```

Comment:
Kind of jarring to have this disappear after uploading a route I feel

Reply:
Yea ur kind of right